### ISSUE TITLE
16.1.8 AppImage source

### ISSUE THREAD

**Issue by beliebie** [FuLygon/Navicat-Linux#4]
Now that the description mentions support for versions up to v16.1.8, are there any official/reliable sources that have the v16.1.8 AppImage archived? I found it through a 'sketchy' source, but was wondering if there is a recommended way to get it, since the official link is many versions ahead.

**Comment by xuewuerduo** [comment]
Not really. I was looking for that

**Comment by FuLygon** [comment]
I found the old version of Navicat 16 AppImage `16.0.11` here https://github.com/nninjia/navicat16-premium-en-aur through [AUR](https://aur.archlinux.org/packages/navicat16-premium-en) package `PKGBUILD` (**RUN AT YOUR OWN RISK**), I also found out that activation status of Navicat stay persisted for latest version, which I guess can be counted as a workaround for latest Navicat 16, just complete the activation process as usual on older version of Navicat 16, once that done, run the latest version of Navicat 16 AppImage

**Comment by beliebie** [comment]
I also found another [AUR repo](https://github.com/everyx/aur-navicat15-premium-cs-appimage) for Navicat 15 that gave [navicat.rainss.cc](https://navicat.rainss.cc/) as its source. This is a website that has many versions of v15 and v16 archived, including v16.1.8 in multiple languages.

Thing is, that page provides the entire keygen in a webbased form and the AppImages are prepatched with a certificate that works with their website. Luckily, you can revert the AppImage to its original state by moving _usr/lib/libcc.so.bak_ to _usr/lib/libcc.so_ (inside the AppImage). That way the AppImages (or their unpacked files) are patchable again so you can use it with the offline keygen and your own certificate. I personally don't like the idea of a website possibly going offline and losing a working patch for new activations.

Their AppImage is working fine and seems legit overall. That being said, newer versions indeed seem to stay activated and see the key, however, my name doesn't get recognized - instead it says 'Not Licensed'. Trial doesn't seem to get triggered though, so I guess it doesn't matter.

I still wish for more official sources/untouched AppImages with a clear source. Too bad websites like archive.org didn't properly archive the Linux versions... Thanks for now!

**Comment by rodentskie** [comment]
+1 on the official link for the `16.1.8` version.

**Comment by carlosmrivera** [comment]
Have you found any reliable source for 16.1.8 version?

**Comment by xuewuerduo** [comment]
> 您是否找到 16.1.8 版本的可靠来源？

No, because I found Navicat Premium Lite 17 that I can use for free！

**Comment by beliebie** [comment]
> 
> No, because I found Navicat Premium Lite 17 that I can use for free！
> 

Thanks so much for this tip! Seems like it's only been out for a few days but this already simply solves all hassle for average users. This is amazing; free, legal, official and with updates.

**Comment by rodentskie** [comment]
Here's a valid version of Navicat Premium 16, [link](https://drive.google.com/drive/folders/1cmxw4OYnzuGed5lOyCFEgdQb5B2Mqi_h?usp=sharing)

Tested it and works fine.

**Comment by beliebie** [comment]
> Here's a valid version of Navicat Premium 16, [link](https://drive.google.com/drive/folders/1cmxw4OYnzuGed5lOyCFEgdQb5B2Mqi_h?usp=sharing)
> 
> Tested it and works fine.

Thank you! Which version of v16 is this? 16.1.8? And who archived this/was this archived directly from the original url or found somewhere else later?

**Comment by houlairen-1** [comment]
It comes from official site.
https://download.navicat.com/download/navicat16-premium-en.AppImage
